mockgcp/mock_http_roundtrip.go

```diff
@@ -59,7 +59,7 @@ import (
 	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mocksecretmanager"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockservicenetworking"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockserviceusage"
-	mockspannerinstance "github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockspanner/admin"
+	mockspanner "github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockspanner/admin"
```

Comment:
+1 thanks!

Reply:
Actually I had a question on this -- Why did you create the `admin/` subdirectory for the instance mock? For other resources it looks like the code is placed directly in `mockgcp/mock<resourcename>/`

Reply:
I'm trying to align that with its cloud client API https://pkg.go.dev/cloud.google.com/go/spanner#section-directories where the `admin/` has instance and database and `executor` has executor. I assume that's how spanner categorize its resources (and maybe will add more resources in the executor dir). But I don't have strong preference on this.